src/main/java/lotto/LottoGroup.java

```diff
@@ -0,0 +1,23 @@
+package lotto;
+
+import java.util.List;
+
+public class LottoGroup {
+	private final List<Lotto> lottoGroup;
+
+	public LottoGroup(List<Lotto> lottoNumbers) {
+		this.lottoGroup = lottoNumbers;
+	}
+
+	public LottoPaper lottoResultReport(Lotto winningNumbers) {
+		LottoPaper lottoReport = new LottoPaper();
+
+		for (int i = 0; i < this.lottoGroup.size(); i++) {
+			Lotto unknownLotto = lottoGroup.get(i);
+			Lotto knownLotto = unknownLotto.unmaskedLotto(winningNumbers);
+			lottoReport = knownLotto.write(lottoReport);
+		}
```

Comment:
foreach문을 사용하셔도 될 것같네요 :-)
```suggestion
		for (Lotto unknownLotto : this.lottoGroup) {
			Lotto knownLotto = unknownLotto.unmaskedLotto(winningNumbers);
			lottoReport = knownLotto.write(lottoReport);
		}
```

Reply:
감사합니다~!